src/config.py

```diff
@@ -301,6 +307,67 @@ def load(self, load_include_dirs=True):
         """
 
         try:
+            PULL_SERVER_SIDE_CONFIG_PARAM = 'True'
+            # Read configuration files from default directories
+            config_files = [self.config_filename]
+            if load_include_dirs:
+                config_files.extend(self._list_configs(self.config_d))
+            self._set_config_file_perms(config_files)
+
+            # Read in json config file
+            with open(config_files[0]) as data_file:
+                d_conf = json.loads(data_file.read())
+            d_configFile = d_conf[CONFIG_PARAM]
+
+            # Get optional user-provided configuration directory
+            self.include = d_configFile.get(INCLUDE_PARAM)
+
+            # Load configuration files from user-provided directory
+            if load_include_dirs and self.include:
+                with open(self._list_configs(self.include)) as config_data:
+                    config_files.extend(json.loads(config_data).read())
+
+            log.log.debug('Configuration files loaded: %s', ', '.join(config_files))
+            self._load_parameters_json(d_configFile)
+            self._configure_proxy_json(d_configFile)
+            new_suppress_ssl = d_configFile.get(SUPPRESS_SSL_PARAM)
+
+            if new_suppress_ssl == 'True':
+                self.suppress_ssl = new_suppress_ssl == 'True'
```

Comment:
Logentries code logic as follows:
Its comparing whether or not new_suppress_ssl is string 'True'. 
If it is set self.suppress_ssl to boolean True. Otherwise is it kept at its default value of boolean False.